index.html

```diff
@@ -3409,6 +3409,44 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+      <section>
+        <h3>Media Types</h3>
+
+        <p>
+          Media types as defined in [[RFC6838]] serve a useful purpose with verifiable 
+          credentials, especially in regards to identifcation of a particular syntax 
+          that is in use with a verifiable credential.
+        </p>
+        <p>
+          Syntaxes SHOULD be identified by a media type, and certain conventions as 
+          outlined in this section should be followed when defining or using media types 
+          with verifiable credentials.
+        </p>
+        <p>
+          Any media types associated with the core data model are listed in the section on
+          <a href="#iana-considerations">IANA Considerations</a> for registration with IANA.
+        </p>
+        <p>
+          When defining a media type for use with verifiable credentials, for instance in a 
+          specification that defines a specific syntax, use of the term `verifiable` implies
+          the presence of a <a href="#proofs-signatures">proof</a> with the credential.  
+          This is also the case if the term is abbreviated, such as in `vc+ld+json`. 
+          Use of media type identified as `verifiable` in this way MUST correspond with use 
+          of one or more proofs with the credential.
+        </p>
+        <p>
+          Use of the term `credential` in a media type related to a syntax of verifiable 
+          credentials without the corresponding use of `verifiable` as noted above does not 
+          imply the presence of a proof with the <a href="#dfn-credential">credential</a>.
+        </p>
+        <p>
+          At the time of this writing, there are two media types associated with the core data 
+          model: `application/credential+ld+json` and `application/verifiable+credential+ld+json`.
```

Comment:
Notably, we haven't actually defined the latter one yet, but this is just an informative thing that can be changed as needed.

Reply:
exactly - especially given the discussion on #1034 re `verifiable-credential` vs `verifiable+credential` this gives us a good way to get aligned

Reply:
@msporny has requested to use `vc+ld+json` elsewhere, which I agree with, but we can leave it like this for now I think.